packages/vue/src/components/organisms/SfAccordion/SfAccordion.vue

```diff
@@ -14,8 +14,16 @@ Vue.component("SfAccordionItem", SfAccordionItem);
 export default {
   name: "SfAccordion",
   props: {
+    /**
+     * Opens an accordion item based on title
+     */
+    open: {
+      type: [String, Array],
+      default: ""
+    },
     /**
      * Opens the first accordion item if set to "true"
+     * @deprecated will be removed in 1.0.0 use open prop instead
```

Comment:
@martaradziszewska can you create [RFC] for section how to deprecated element in our documentation?

Reply:
yes I'll do it :)